content/post/104-dht-deep-dive.md

```diff
@@ -0,0 +1,154 @@
+---
+date: 2020-07-20
+url: /2020-07-20-dht-deep-dive/
+title: "IPFS 0.5 Content Routing Improvements: Deep Dive"
+author: Adin Schmahmann
+header_image: 104-dht-deep-dive.png
+tags: go-ipfs, 0.5 release
+---
+
+<p style="max-width:1000px;margin-left:auto;margin-right:auto;">
+  <img src="/header_images/104-dht-deep-dive.png">
+</p>
+
+At the end of April, we released our largest update to go-ipfs to date: [IPFS 0.5](https://blog.ipfs.io/2020-04-28-go-ipfs-0-5-0/). While there have been many improvements, the changes to IPFS’s Distributed Hash Table (DHT) were especially critical to improving the performance and stability of finding data in IPFS. For some background on how we arrived at the most recent set of DHT changes, take a look at [The Road to the New DHT](https://blog.ipfs.io/2020-05-19-road-to-dht/).
```

Comment:
I know we mention upgrading at the end, but I think the intro is also a good place to plug upgrading.

```suggestion
At the end of April, we released our largest update to go-ipfs to date: [IPFS 0.5](https://blog.ipfs.io/2020-04-28-go-ipfs-0-5-0/). While there have been many improvements, the changes to IPFS’s Distributed Hash Table (DHT) were especially critical to improving the performance and stability of finding data in IPFS. For some background on how we arrived at the most recent set of DHT changes, take a look at [The Road to the New DHT](https://blog.ipfs.io/2020-05-19-road-to-dht/), or try them out yourself in the [latest version of go-ipfs](https://dist.ipfs.io/#go-ipfs).
```